org-ql.el

```diff
@@ -1969,6 +1974,24 @@ ignored."
   :body
   (org-ql--predicate-ts :from from :to to :regexp org-ql-clock-regexp :match-group 1))
 
+(org-ql-defpred log-done (&key from to _on)
```

Comment:
The predicate should probably be named something like `logbook-state` so it could be more flexible.